frontend/pages/createorganization.js

```diff
@@ -344,6 +344,8 @@ export default function CreateOrganization({ tagOptions, rolesOptions, allHubs }
         textKey: "info.short_description",
         rows: 5,
         headlineTextKey: "short_description",
+        maxCharacters: 280,
```

Comment:
I'd allow a bit more characters for the translation, maybe 350 because some languages make the text longer